GVFS/GVFS.Common/Maintenance/LooseObjectsStep.cs

```diff
@@ -22,26 +23,63 @@ public LooseObjectsStep(GVFSContext context, bool requireCacheLock, bool forceRu
 
         public override string Area => nameof(LooseObjectsStep);
 
+        public virtual int MaxLooseObjectsInPack => 1000;

```

Comment:
Some math to do here. How long does it take to pack 1000 objects? 10,000? 100,000? If we are running this only once a week, then we probably want this to be _at least_ 100,000 due to over-hydrating causes 2.1 million blob downloads. Unless there is a reason why that would be too many objects.

Reply:
Agreed. Also consider the size of a typical hourly prefetch pack or a daily prefetch pack. How many objects do we see there?

Reply:
Keep in mind that the daily/hourly prefetch packs contain commits and trees, and we expect this step to create packs from blobs. The blobs will be much bigger.

Reply:
Fair point. Either way I think 1000 is too small by an order of magnitude or two.

Reply:
Went with 100,000 for now